pycoal/mineral.py

```diff
@@ -24,6 +24,42 @@
 import time
 import fnmatch
 import shutil
+from pycoal import mineral
+import multiprocessing
+from joblib import Parallel, delayed
+
+
+def CalculatePixel(pixel, x, y, classified, library, threshold, resample, scores_file_name):
```

Comment:
Provide documentation for the function. An example can be found [here](https://github.com/capstone-coal/pycoal/blob/master/pycoal/mineral.py#L39-L66).

Reply:
Also, let think about whether `CalculatePixel` is a suitable name for the function. It appears to me it is _**calculating a confidence value of a classifed pixel**_?